core/src/main/kotlin/io/customer/android/sdk/CustomerIO.kt

```diff
@@ -0,0 +1,79 @@
+package io.customer.android.sdk
+
+import androidx.annotation.VisibleForTesting
+import io.customer.android.core.di.SDKComponent
+import io.customer.base.internal.InternalCustomerIOApi
+
+// TODO: Move this class and its dependencies (CustomerIOInstance) to the correct package.
+// We need to move this class to the right package to avoid breaking imports for the users of the SDK.
+// We have placed the class in the wrong package for now to avoid breaking the build.
+// Once old implementations are removed, we can move the class to the correct package.
```

Comment:
The note is reminder to what I explained in PR description.

Reply:
> We need to move this class to the right package to avoid breaking imports for the users of the SDK.

We are going with a breaking change, this was part of what import previously? and what will be the new one?

Reply:
> We are going with a breaking change

Yes, but this does mean we want to break everything that worked before 😅 We should update the package only if we really want to, so I called it out make sure that's by choice and not forced.

> This was part of what import previously? and what will be the new one?

Sorry I didn't get the question here. Are you asking what was the import of `CustomerIO` before and after? If yes, it will be `io.customer.sdk` for both.